### PR TITLE
GOV 285: New g2p-sandbox-security chart 

### PR DESCRIPTION
**Pull request by danishjamal104**
## Description

* Created new chart and new circle ci step for deploying security chart.

[GOV-285](https://mifosforge.jira.com/browse/GOV-285)

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Design related bullet points or design document link related to this PR added in the description above. 

- [x] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [x] Create/update unit or integration tests for verifying the changes made.

- [x] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [x] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

[GOV-285]: https://mifosforge.jira.com/browse/GOV-285?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: ph-ee-g2psandbox-0.0.0.tgz
Comment (fynmanoj) [inline ph-ee-g2psandbox-0.0.0.tgz:1]: remove binaries/ update git ignore

File: helm/g2p-sandbox-security/README.md
Comment (fynmanoj) [inline helm/g2p-sandbox-security/README.md:1]: update readme with the steps to install/ setup ingress

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: ph-ee-g2psandbox-0.0.0.tgz
Comment (danishjamal104) [reply ph-ee-g2psandbox-0.0.0.tgz:1]: done

File: helm/g2p-sandbox-security/README.md
Comment (danishjamal104) [reply helm/g2p-sandbox-security/README.md:1]: done

**Review by fynmanoj — APPROVED** [review]

**Pull request merged**